File: app/src/main/java/it/cammino/risuscito/database/entities/Cronologia.java
package it.cammino.risuscito.database.entities;

import android.arch.persistence.room.Entity;
import android.arch.persistence.room.PrimaryKey;

import java.sql.Date;

@Entity
public class Cronologia {

    @PrimaryKey
    public int idCanto;

    public Date ultimaVisita = new Date(System.currentTimeMillis());

}

File: app/src/main/java/it/cammino/risuscito/database/entities/Cronologia.kt
package it.cammino.risuscito.database.entities

import androidx.room.Entity
import androidx.room.PrimaryKey

import java.sql.Date

@Entity
class Cronologia {

    @PrimaryKey
    var idCanto: Int = 0

    var ultimaVisita = Date(System.currentTimeMillis())

}
